client/src/components/Header.js

```diff
@@ -13,6 +16,67 @@ const Header = () => {
       </nav>
     </div>
   );
+
 };
 
+class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      auth: '',
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get('auth/current_user')
+      .then(user => {
+        this.setState({ auth: user.data });
+        console.log(user);
+      })
+      .catch(err => {
+        console.log(err);
```

Comment:
in case of an error it should show something on the page instead of console logging the error, it's fairly easy and not a big task

Reply:
Done